django_redshift_backend/base.py

```diff
@@ -160,8 +166,8 @@ def create_model(self, model):
             params.extend(extra_params)
             # FK
             if field.remote_field and field.db_constraint:
-                to_table = field.remote_field.to._meta.db_table
-                to_column = field.remote_field.to._meta.get_field(
+                to_table = field.remote_field.related_model._meta.db_table
```

Comment:
Its bug to migrate django 2.0... Sorry

https://github.com/shimizukawa/django-redshift-backend/pull/43

Reply:
#43 has been merged. @benjyw  please rebase this PR if you have chance.